internal/controllers/core/uibutton/reconciler.go

```diff
@@ -4,28 +4,31 @@ import (
 	"context"
 	"fmt"
 
-	"sigs.k8s.io/controller-runtime/pkg/builder"
-
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
 	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	"github.com/tilt-dev/tilt/internal/hud/server"
+	"github.com/tilt-dev/tilt/internal/store"
+	"github.com/tilt-dev/tilt/internal/store/uibuttons"
 	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
 )
 
 type Reconciler struct {
 	client ctrlclient.Client
 	wsList *server.WebsocketList
+	store  store.RStore
```

Comment:
FYI there's a `store.Dispatcher` iface that's bound with wire - not critical, but I've been using it to avoid the temptation for a reconciler to start actually mutating the store 🤪 

https://github.com/tilt-dev/tilt/blob/449fe2dbcc77c7bfacea82d1b0e14f93c4b0194a/internal/store/logger.go#L71-L73